internal/proxy/validate_util.go

```diff
@@ -934,3 +935,25 @@ func newValidateUtil(opts ...validateOption) *validateUtil {
 
 	return v
 }
+
+func ValidateAutoIndexMmapConfig(isVectorField bool, typeParams map[string]string) error {
+	if !Params.AutoIndexConfig.Enable.GetAsBool() {
```

Comment:
if autoindex not enabled, what key is user actually using?
mmap.index or mmap.data?

Actually I sugges to still use mmap.enabled, the difference is only only why this key is applied on data or index?